modules/kubernetes/pages/k8s-operator/index.adoc

```diff
@@ -7,35 +7,32 @@ It enables you to automate operations such as the creation, status checking, res
 
 IMPORTANT: Kubernetes Operator support is currently a Preview Feature. Preview Features give users an early look at future production-level features. Preview Features should not be used for production deployments.
 
-NOTE: Additionally, we have provided a public GitHub repo (https://github.com/tigergraph/ecosys/tree/master/k8s) to our preview documentation of the Kubernetes Operator. These docs are frequently updated to support operation, troubleshooting, and all other relevant use cases.
+NOTE: Additionally, we have provided a public xref:https://github.com/tigergraph/ecosys/tree/master/k8s[GitHub Repo] to our preview documentation of the Kubernetes Operator. These docs are frequently updated to support operation, troubleshooting, and all other relevant use cases.
```

Comment:
Should we be saying "additionally"?  Is GitHub the only documentation they have for now?

Reply:
Good question. I changed this with the latest commit.